app-editors/kak_lsp/kak_lsp-12.1.0.recipe

```diff
@@ -0,0 +1,447 @@
+CHECKSUM_SHA256_108="f14ee04d9415b52b3aeab06258a3f07093182b88ba0f9b8d203f211a7a7d41c7"
+
+SOURCE_URI_109="https://static.crates.io/crates/url/url-2.2.2.crate"
+CHECKSUM_SHA256_109="a507c383b2d33b5fc35d1861e77e6b383d158b2da5e14fe51b83dfedf6fd578c"
+
+SOURCE_URI_110="https://static.crates.io/crates/vec_map/vec_map-0.8.2.crate"
+CHECKSUM_SHA256_110="f1bddf1187be692e79c5ffeab891132dfb0f236ed36a43c7ed39f1165ee20191"
+
+SOURCE_URI_111="https://static.crates.io/crates/version_check/version_check-0.9.3.crate"
+CHECKSUM_SHA256_111="5fecdca9a5291cc2b8dcf7dc02453fee791a280f3743cb0905f8822ae463b3fe"
+
+SOURCE_URI_112="https://static.crates.io/crates/wasi/wasi-0.10.2+wasi-snapshot-preview1.crate"
+CHECKSUM_SHA256_112="fd6fbd9a79829dd1ad0cc20627bf1ed606756a7f77edff7b66b7064f9cb327c6"
+
+SOURCE_URI_113="https://static.crates.io/crates/wasm-bindgen/wasm-bindgen-0.2.77.crate"
+CHECKSUM_SHA256_113="5e68338db6becec24d3c7977b5bf8a48be992c934b5d07177e3931f5dc9b076c"
+
+SOURCE_URI_114="https://static.crates.io/crates/wasm-bindgen-backend/wasm-bindgen-backend-0.2.77.crate"
+CHECKSUM_SHA256_114="f34c405b4f0658583dba0c1c7c9b694f3cac32655db463b56c254a1c75269523"
+
+SOURCE_URI_115="https://static.crates.io/crates/wasm-bindgen-macro/wasm-bindgen-macro-0.2.77.crate"
+CHECKSUM_SHA256_115="b9d5a6580be83b19dc570a8f9c324251687ab2184e57086f71625feb57ec77c8"
+
+SOURCE_URI_116="https://static.crates.io/crates/wasm-bindgen-macro-support/wasm-bindgen-macro-support-0.2.77.crate"
+CHECKSUM_SHA256_116="e3775a030dc6f5a0afd8a84981a21cc92a781eb429acef9ecce476d0c9113e92"
+
+SOURCE_URI_117="https://static.crates.io/crates/wasm-bindgen-shared/wasm-bindgen-shared-0.2.77.crate"
+CHECKSUM_SHA256_117="c279e376c7a8e8752a8f1eaa35b7b0bee6bb9fb0cdacfa97cc3f1f289c87e2b4"
+
+SOURCE_URI_118="https://static.crates.io/crates/web-sys/web-sys-0.3.54.crate"
+CHECKSUM_SHA256_118="0a84d70d1ec7d2da2d26a5bd78f4bca1b8c3254805363ce743b7a05bc30d195a"
+
+SOURCE_URI_119="https://static.crates.io/crates/whoami/whoami-1.1.3.crate"
+CHECKSUM_SHA256_119="f7741161a40200a867c96dfa5574544efa4178cf4c8f770b62dd1cc0362d7ae1"
+
+SOURCE_URI_120="https://static.crates.io/crates/winapi/winapi-0.3.9.crate"
+CHECKSUM_SHA256_120="5c839a674fcd7a98952e593242ea400abe93992746761e38641405d28b00f419"
+
+SOURCE_URI_121="https://static.crates.io/crates/winapi-i686-pc-windows-gnu/winapi-i686-pc-windows-gnu-0.4.0.crate"
+CHECKSUM_SHA256_121="ac3b87c63620426dd9b991e5ce0329eff545bccbbb34f3be09ff6fb6ab51b7b6"
+
+SOURCE_URI_122="https://static.crates.io/crates/winapi-x86_64-pc-windows-gnu/winapi-x86_64-pc-windows-gnu-0.4.0.crate"
+CHECKSUM_SHA256_122="712e227841d057c1ee1cd2fb22fa7e5a5461ae8e48fa2ca79ec42cfc1931183f"
+
+ARCHITECTURES="all !x86_gcc2"
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	kak_lsp$secondaryArchSuffix = $portVersion
+	cmd:kak_lsp
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	kakoune$secondaryArchSuffix
```

Comment:
It's the other way around, kakoune can have an optional dependency for kak-lsp, no need for this here ;)
```diff
- 	kakoune$secondaryArchSuffix
```

Reply:
kak-lsp is useless without kakoune. On Arch kakoune optionally depends on kak-lsp and kak-lsp depends on kakoune.

Reply:
My bad there, didn't check on this one ;) So let's leave it in (build in progress here atm)

Reply:
How did the build go?

Reply:
if you want kak-lsp to be proposed to install along kakoune:
`SUPPLEMENTS="kakoune$secondaryArchSuffix"`

Reply:
Build went OK, now I remember why I don't like rust packages to build :)